File: Intermediate/algorithms/PHP/binary_search.php
<?php 
  
function binarySearch($arr, $v) 
{ 
    if (count($arr) === 0) return false; 
    $low = 0; 
    $high = count($arr) - 1; 
      
    while ($low <= $high) { 

        $mid = floor(($low + $high) / 2); 

        if($arr[$mid] == $v) { 
            return true; 
        } 
  
        if ($v < $arr[$mid]) { 
            $high = $mid -1; 
        } 
        else { 
            $low = $mid + 1; 
        } 
    } 
    return false; 
} 
  
// Usage
$arr = array(1, 2, 3, 4, 5); 
$value = 5; 
if (binarySearch($arr, $value) == true) { 
    echo $value . " exists"; 
} 
else { 
    echo $value . " Doesn't exist"; 
}

File: Intermediate/algorithms/python/binary_search.py
def binarySearch(Number, SortedList):
      Low = 0
      High = len(SortedList) - 1
      while Low <= High :
            Mid = (High + Low) // 2
            if SortedList[Mid] == Number:
                  return Mid
            elif SortedList[Mid] > Number :
                  High=Mid - 1
            else :
                   Low = Mid + 1

def main():
      my_list = [1,2,30,46,58,69,74,83,90]
      my_list = sorted(my_list)
      print ("This is my array:", my_list)
      number=input("What number are you looking for? ")
      number = int(number)
      Sort = binarySearch(number, my_list)
      print ("The position in the array is:",Sort)

if __name__ == "__main__":
      main()